src/websockets/api.js

```diff
@@ -0,0 +1,218 @@
+const jwt = require('jsonwebtoken');
+const moment = require('moment');
+const { createClient } = require('redis');
+const config = require('../config');
+const logger = require('../startup/logger');
+
+export const getUserConnectionKey = userId => `${userId}-connections`;
+
+export const TIMER_UPDATES_CHANNEL = 'timer-service-updates';
+
+export const redisClients = {
+  main: createClient({
+    url: process.env.REDIS_APP_URL,
+  }),
+  publisher: createClient({
+    url: process.env.REDIS_APP_URL,
+  }),
+  subscriber: createClient({
+    url: process.env.REDIS_APP_URL,
```

Comment:
Should main, publisher, and subscriber have the same url?

Reply:
Yes, this is all part of redis